addon/components/uni-radio-button.js

```diff
@@ -18,5 +18,17 @@ export default Component.extend({
 
   change() {
     this.get('hasChanged')(this.get('value'));
+  },
+
+  actions: {
+    onClick(value) {
+      if (isPresent(this.get('onClick'))) {
+        this.get('onClick')();
+
+        return false;
+      }
+
+      this.set('value', value);
```

Comment:
Have you tested if the parameter sent to `onClick` is the value? In js events, it is normally the event object. Just to be sure